### DISCUSSION TITLE
Meaning of peak matrix value

### DISCUSSION THREAD

**Discussion by chancxz** [GreenleafLab/ArchR#1907] [Questions / Documentation] [answered]
Hi.
First, thanks for awesome tool(ArchR) for analyzing ATAC-seq.

When I get peak matrix from `pm <- getMatrixFromProject(proj1, useMatrix='PeakMatrix')`, I wonder what is a meaning of value in matrix.
In my data, the values of matrix are in ranges of 0-4. 
In my opinion row names are peak ranges and column names are cell barcodes, therefore the value should be 0 or 1 for representing there is a peak or not.

I searched for a lot to figure this out, but there is no meaning about peak matrix value in tutorial or github. 
I wonder what is meaning of peak matrix value, and why they are 0 to 4.

Best regards,
Chan

**Comment by rcorces** [comment, marked as answer]
the values are the number of insertions observed per cell in the given peak region

**Reply by chancxz** [reply]
Thank you for answer! Helped a lot!!
